src/styles/site/fileManager/FyLandingFileManagerStyles.module.css

```
@@ -14,7 +14,342 @@
+    height: 38px;
+    flex: 0 0 38px;
+    border-radius: 11px;
+    line-height: 0;
+}
+
+.iconChip::before {
+    content: "";
+    position: absolute;
+    inset: 0;
+    border-radius: inherit;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)